.travis.yml

```diff
@@ -1,34 +1,77 @@
 ---
 language: node_js
-dist: trusty
-node_js:
-  - "stable"
-
+node_js: stable
 sudo: false
+dist: trusty
 
 addons:
   chrome: stable
 
 cache:
   directories:
     - $HOME/.npm
-    - $HOME/.cache  # includes bowers cache
-    - node_modules
 
-matrix:
-  fast_finish: true
+env:
+  global:
+    - JOBS=1
```

Comment:
Add the env var `CI=true`, it disables the disk cache on CI environments.
[Link to CI](https://github.com/stefanpenner/broccoli-persistent-filter#persistent-cache)

Reply:
There is no need for that. Travis has it has default environment variable:

https://docs.travis-ci.com/user/environment-variables/#default-environment-variables